sdk/communication/azure-communication-phonenumbers/test/_shared/utils.py

```diff
@@ -4,15 +4,24 @@
 # license information.
 # -------------------------------------------------------------------------
 
-from .fake_token_credential import FakeTokenCredential
-from azure.identity import DefaultAzureCredential
 from azure.core.pipeline.policies import HttpLoggingPolicy
 
 def create_token_credential():
     # type: () -> FakeTokenCredential or DefaultAzureCredential
     from devtools_testutils import is_live
     if not is_live():
+        from .fake_token_credential import FakeTokenCredential
         return FakeTokenCredential()
+    from azure.identity import DefaultAzureCredential
+    return DefaultAzureCredential()
+
+def async_create_token_credential():
```

Comment:
We should modify all the shared folders across all packages to have the changes you did to the one for phone numbers as well

Reply:
I'm sorry about the comment. I thought the modified file was in the under source shared folder and not the test one. You can completely ignore the previous comment 👍🏻